meta-zarhus-webkit/recipes-networking/cog/cog_%.bbappend

```diff
@@ -0,0 +1,14 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI:append = " \
+    file://cog-autostart.service \
+    "
+
+inherit systemd
+
+SYSTEMD_AUTO_ENABLE = "enable"
```

Comment:
I don't think we want cog to autostart in each image by default. It should be disabled by default and only enabled if needed.